src/Pos/Block/Logic.hs

```diff
@@ -751,16 +750,18 @@ createMainBlockPure
     -> SscPayload ssc
     -> UpdatePayload
     -> SecretKey
-    -> MainBlock ssc
+    -> m (MainBlock ssc)
 createMainBlockPure limit prevHeader txs pSk sId psks sscData usPayload sk =
-    flip evalState limit $ do
+    flip evalStateT limit $ do
```

Comment:
Lool, how it happened that `flip evalState` is actually shorter than shortcut introduced for it 😄